ansible/roles/stack-sunbird/templates/analytics_api_service.conf

```diff
@@ -219,3 +221,8 @@ device.api.enable.debug.log=true
 kafka.device.register.topic={{ env_name }}.events.deviceprofile
 kafka.metrics.event.topic={{ env_name }}.pipeline_metrics
 kafka.broker.list="{{ ingestion_kafka_brokers }}"
+
+cdn.host="https://data.{{proxy_server_name}}/datasets"
+public.data_exhaust.datasets=["public-data-exhaust"]
+public.data_exhaust.expiryMonths=6
+public.data_exhaust.max.interval.days=30
```

Comment:
can we variablize the days as well so that the adopters need not fork repo to update this configuration.